mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -129,86 +164,53 @@ export class ExperimentListView extends Component {
               title='Hide experiment list'
-              className='collapser fa fa-chevron-left login-icon'
             />
           </div>
           <Input
-            className='experiment-list-search-input'
-            type='text'
             placeholder='Search Experiments'
             aria-label='search experiments'
             value={searchInput}
             onChange={this.handleSearchInputChange}
           />
-          <div className='experiment-list-container' style={{ height: experimentListHeight }}>
-            {this.props.experiments
-              // filter experiments based on searchInput
-              .filter((exp) =>
-                exp
-                  .getName()
-                  .toLowerCase()
-                  .includes(searchInput.toLowerCase()),
-              )
-              .map((exp, idx) => {
-                const { name, experiment_id } = exp;
-                const active =
-                  this.props.activeExperimentId !== undefined
-                    ? experiment_id === this.props.activeExperimentId
-                    : idx === 0;
-                const className = `experiment-list-item ${
-                  active ? 'active-experiment-list-item' : ''
-                }`;
-                return (
-                  <div key={experiment_id} title={name} className={`header-container ${className}`}>
-                    <Link
-                      style={{ textDecoration: 'none', color: 'unset', width: '80%' }}
-                      to={Routes.getExperimentPageRoute(experiment_id)}
-                      onClick={active ? (ev) => ev.preventDefault() : (ev) => ev}
-                    >
-                      <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>{name}</div>
-                    </Link>
-                    {/* Edit/Rename Experiment Option */}
-                    <IconButton
-                      icon={<EditOutlined />}
-                      onClick={this.handleRenameExperiment}
-                      data-experimentid={experiment_id}
-                      data-experimentname={name}
-                      style={{ marginRight: 10 }}
-                    />
-                    {/* Delete Experiment option */}
-                    <IconButton
-                      icon={<i className='far fa-trash-alt' />}
-                      onClick={this.handleDeleteExperiment}
-                      data-experimentid={experiment_id}
-                      data-experimentname={name}
-                      style={{ marginRight: 10 }}
-                    />
-                  </div>
-                );
-              })}
+          <div className='experiment-list-container'>
+            <Tree
+              treeData={treeData}
+              dangerouslySetAntdProps={{
+                selectable: true,
+                multiple: true,
+                selectedKeys: [activeExperimentId],
+                titleRender: this.renderListItem,
```

Comment:
To support multi-experiment run comparison, we can add

```suggestion
                checkable: true,
                selectable: true,
                multiple: true,
                onCheck: this.handleClick,
                selectedKeys: [activeExperimentId],
                titleRender: this.renderListItem,
```

Reply:
Quick demo:

https://user-images.githubusercontent.com/17039389/167361741-d134ab40-11d0-4faa-8dc6-27d20c5c578b.mov